tests/conftest.py

```diff
@@ -243,6 +253,26 @@
 )
 
 
+TEST_IPFS_REGISTRY_CONFIG: Dict = {
+    "registry_config": {
+        "default": REGISTRY_REMOTE,
+        "settings": {
+            REGISTRY_REMOTE: {
+                "default": REMOTE_IPFS,
+                REMOTE_HTTP: {
+                    AUTH_TOKEN_KEY: "some key",  # auth token for registry
+                    "registry_api_url": "https://agents-registry.prod.fetch-ai.com/api/v1",  # registry url
+                },
+                REMOTE_IPFS: {
+                    "ipfs_node": "/dns/registry.autonolas.tech/tcp/443/https"
```

Comment:
We shouldn't be using the production node for tests, we have support for using the IPFS daemon using a fixture, please open a ticket for using the IPFS daemon for IPFS registry tests

Reply:
integration tests needs a real node with data uploaded to
anyway not too hard to mock